packages/node_modules/@webex/plugin-meetings/src/metrics/constants.js

```diff
@@ -51,7 +51,11 @@ const BEHAVIORAL_METRICS = {
   ADHOC_MEETING_FAILURE: 'js_sdk_adhoc_meeting_failure',
   VERIFY_PASSWORD_SUCCESS: 'js_sdk_verify_password_success',
   VERIFY_PASSWORD_ERROR: 'js_sdk_verify_password_error',
-  VERIFY_CAPTCHA_ERROR: 'js_sdk_verify_captcha_error'
+  VERIFY_CAPTCHA_ERROR: 'js_sdk_verify_captcha_error',
+  MOVE_TO_SUCCESS: 'js_sdk_move_to_success',
+  MOVE_TO_FAILURE: 'js_sdk_move_to_failure',
+  MOVE_FROM_SUCCESS: 'js_sdk_move_FROM_success',
```

Comment:
'js_sdk_move_FROM_success' could be 'js_sdk_move_from_success' & 
'js_sdk_move_FROM_failure' could be 'js_sdk_move_from_failure'

Reply:
done